### ISSUE TITLE
Export creating no files on system

### ISSUE THREAD

**Issue by HalfBlu3** [SonicMastr/godot-vita#35]
**Godot version:**
3.5.cr5
**OS/device including version:**
Windows 11
**Issue description:**
When I try to export my game, nothing get output. I can make a .pck or .zip, but not a .vpk. When I click export all -> release, the packing progress bar goes to 92-97%, but the exporting bar stays at 0% and then the export window closes. When I check the export location there's nothing there. Export project also goes to like 92-97% before closing with no output.
**Minimal reproduction project:**
Brotato on steam with Steam functionality stripped out.

**Comment by HalfBlu3** [comment]
Figured it out. It can only export to certain folders for whatever reason. To make sure it will work, make a dummy folder and put it in the destination. When the prompt comes up, if you don't see the dummy folder it can't "read" the folder and won't export successfully.

**Comment by HalfBlu3** [comment]
Yeah, I ended up reinstalling like twice and then it just worked. No idea why. Never got the game functionally ported, but the .vpk would be exported properly and did run

**Comment by mertyagmur** [comment]
@HalfBlu3 I actually got the game running. I initially got stuck on godot screen as well. The reason that it wasn't getting past that is because the title screen uses more vram than vita has. I removed all the animated sprites from the title screen and it worked. The performance is not perfect but it is very playable. I'm experimenting with compressing the assets for better performance. After that I'm planning to work on a guide or something for the community on how to get it working.

**Comment by HalfBlu3** [comment]
Oh, that's awesome! I was considering starting up the project again if you want any help. Out of curiosity, how did you figure out it was a vram issue? Was it just trial and error?

**Comment by mertyagmur** [comment]
@HalfBlu3 So I was using PrincessLog for logging and saw some error messages related to memory. They were not very clear though so I looked into the monitors section of godot's debugger. I observed the vram usage there and noticed that even though the game usually uses around 50-60MB of vram, in the title screen it goes above 100MB. I was able to get it down to the usual 50-60MBs by removing some of the assets from the title screen and the game was able to start.

**Comment by HalfBlu3** [comment]
hmm, I didn't know there were log programs even available on the vita lol. Guess that's what happens when I don't have the proper time to deep dive. That's a surprisingly simple fix. Good luck finishing the project!

**Comment by kultuk** [comment]
> Figured it out. It can only export to certain folders for whatever reason. To make sure it will work, make a dummy folder and put it in the destination. When the prompt comes up, if you don't see the dummy folder it can't "read" the folder and won't export successfully.

What do you mean by a dummy folder? I tried several locations and non worked
